src/fidesops/schemas/redis_cache.py

```diff
@@ -1,10 +1,16 @@
 from typing import Optional
+from pydantic import Extra
 
 from fidesops.schemas.base_class import BaseSchema
 
 
 class PrivacyRequestIdentity(BaseSchema):
     """Some PII grouping pertaining to a human"""
 
-    phone_number: Optional[str]
-    email: Optional[str]
+    phone_number: Optional[str] = None
+    email: Optional[str] = None
+
+    class Config:
+        """Only allow phone_number and email to be supplied"""
+
+        extra = Extra.forbid
```

Comment:
In theory we should be able to match any labelled data in a privacy request, as long as it matches the key looked for in the data configuration. Is it a requirement that we're limiting it?

Reply:
I think it's a requirement, I believe @seanpreston told me there had been a decision to just start with supporting `phone_number` and `email` identities - correct me if I'm wrong Sean.

It's worth pointing out that this doesn't change behavior.  The `PrivacyRequestIdentity` schema _already_ restricts that only phone_number and email identities are cached, this change just keeps the user from _attempting to submit_ other fields.  Otherwise, the user can submit a street_address, and believe we are adding it to the identity graph, but we are not.

Reply:
I don't think that makes sense (I'd missed that previously in the privacy request schema) - what does it add to disallow matching on e.g. social security number or any other field(s) someone might want to drive this with? That being said, agree that it's a pre-existing product decision.

Reply:
I agree we're going to have to add support for other fields, but I guess we'll intentionally add them over time?